client/screens/HomePage.js

```diff
@@ -25,8 +34,15 @@ const Home = (props) => {
 
   const cardSwipeRef = useRef(null);
 
+  const generateMatchId = (id1, id2) => (id1 > id2 ? id1 + id2 : id2 + id1);
```

Comment:
Are these ids strings or numbers?  If they're numbers, you could end up with a non-one-to-one mapping.

Reply:
Actually, do these ids need to be unique?  If they're strings, you will receive the same match id from the following example:

`("1", "112") -> "1112"`
`("11","12") -> "1112"`

Reply:
The ids are strings. I'm still not sure if this was necessary, I just was thinking in terms of consistency for matched users id to add one first before the other always and have that as the id for matched users document

`("OltIf8", "skVutJ") -> "skVutJOltIf8"`
`("skVutJ","OltIf8") -> "skVutJOltIf8"`

Maybe I overthought/overdid this but my thought process was since every user id is a unique string, adding both users ids together or concatenating them ,to create a new unique id as the matchedId would be useful for the messaging portion but to be honest I'm still trying to figure that out

Reply:
I think the best way to go about it would be to edit your DB schema a bit to add a `matches` table.  You'd probably want the following columns: `id` (primary key), `matchRequester` (the id of whoever initiated the match request, first person to swipe right), `matchRequestee` (the id of whoever was swiped right on), `accepted` (a bool indicating whether or not the requestee accepted the match or not).  You can then query this table for information on matches.

Reply:
Thank you! Definitely going to look into this more today!